staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -106,6 +108,21 @@ var baseOpts = []VersionedOptions{
 			ext.Strings(ext.StringsVersion(2)),
 		},
 	},
+	// Set library
+	{
+		IntroducedVersion: version.MajorMinor(1, 29),
+		EnvOptions: []cel.EnvOption{
+			ext.Sets(),
```

Comment:
Did we not expose any of the sets functions before?

Do we need to update our CEL docs (https://kubernetes.io/docs/reference/using-api/cel/#cel-community-libraries) with the sets library (or would we update that in 1.30 docs)? Is there a tracking issue for updating docs?

Reply:
No we didn't expose the sets func before. And YES we need to update our cel doc in 1.30(it is currently in dark mode). I will create an tracking issue for updating docs here: https://github.com/kubernetes/website/issues/43747